### ISSUE TITLE
Front End: e2e test coverage for VA Police pages

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#16303]
## User Story or Problem Statement
Remaining test coverage from #16208  for the new VA Police data pages on VAMC facilities.

## Acceptance Criteria
- e2e test is completed: When pages are created for systems. Check any system page that side panel element is created and that page exists (this can be done only after skeleton is created and graphQL is changed to pull in side panel link)
- unit tests are completed:
  ~~- CONTENT OF SELECTOR TESTING: (requires: system-facility data from view, mock data for unit test of selector). Test non-existent facility. Test existing facility and expect mock data.~~ As of sprint planning 12/6, it was decided that the selector is out of scope for MVP. Facility contact information will come from Drupal and be listed on the system page.

**Comment by eselkin** [comment]
This is a 2, but blockers will be the VA Police Page being created and the widget being created.

**Comment by maxx1128** [comment]
I would estimate this at a **3.** Adding to existing unit tests is easy, and e2e tests are more complicated to set up but they're also only testing fundamentals like page presence and side navigation. Eli is more familiar with this ticket, however, so I'm okay with going to a 2 if it's their estimate.

**Comment by jilladams** [comment]
Rewrite from police data to police pages since we don't have data / all we have is pages.

**Comment by jilladams** [comment]
Can build tests against content-build branch in progress: [VACMS-16278-VA-Police-Query-And-Template](https://github.com/department-of-veterans-affairs/content-build/tree/VACMS-16278-VA-Police-Query-And-Template)

**Comment by jilladams** [comment]
@eselkin to send notes on which Tugboat to use in order to get test data.

**Comment by eselkin** [comment]
I've been wracking my brain about this all day. The theory seems simple, open a page and test that it's there. 

However, the current e2e tests on content-build test for things that don't have to be published in order to see. For police, pages have to be published for the link to display and the page to render. But we don't want to publish content until there's a "go ahead" which itself includes a requirement testing. So we are at an impasse.

We can do the unit rendered HTML test (I built one). However this isn't the same as testing a rendered page because the components don't get built or brought in (e.g. FAQ, etc.).

**Comment by jilladams** [comment]
@eselkin did we address your concern above in scrum today? I think so but want to confirm. (That discussion: we can ship the test after launch, flag with Collab Cycle that that's how it will work, and have it drafted based on local published content that'll publish on launch, for merge after launch.)

**Comment by eselkin** [comment]
Yep @jilladams addressed

**Comment by eselkin** [comment]
Unit tests built, but e2e has to wait till pages are accessible on prod so cypress can access the pages.

**Comment by jilladams** [comment]
For end of sprint: PR needs code review in order to merge.

**Comment by jilladams** [comment]
Merged for several days, not tagged as flaky. Closing.
